deploy/local/dal

```diff
@@ -84,6 +84,9 @@ set daal_target_arch         "intel64"
 module-whatis "Intel(R) Data Analytics Library for $daal_target_arch."
 
 # Setup environment variables
+setenv          DAL_MAJOR_BINARY   1
+setenv          DAL_MINOR_BINARY   0
```

Comment:
Much better is to specify values for variables in separate text file in following form:
var1=value1
var2=value2
and use this file to generate code to set environment variables for every OS.

For example, on Linux to do so, you can use sed/awk/... to perform required transformations and use `source <(cat version.txt | <transformations>)` to do actual set of env.variables.

Reply:
yes, and ideally this should be taken from single file with version for entire product and not separate version.txt file. 

But this is work for @Alexsandruss that will be completed after release .  For now i would like to expose variables for users in env

Reply:
OK